### PR TITLE
Update geo-commodities.ttl

### LINKED ISSUE #260: Addition to commodities vocabs
Please add following commodities into the vocabs:

1. Geothermal
2. Carbon geostorage / sequestration.

Thank you.

### PR DESCRIPTION
**Pull request by KellyVance**
Closes #260 

**Reviewers**
Tech @LukeHauck 
SME @talebib 
Vocab Owner @SalEdwards01 

**Changes**
- Addition: _Undefined_ for undefined commodities for exploration activities in prospective areas.
- Re-Label: _Coal Seam Gas_ now prefLabel | _Coal Bed Methane_ now altLabel
- Redefine: _Reservoir gas_ to clastic and carbonate reservoirs (differentiate from CSG and shale o&g)
- Additions: Shale Gas and Shale Oil
- Addition: _Water_ as a commodity
- Addition: _Energy Resources_ as non-material resources in which the energy is the commodity
   - Addition: Energy Resource child concepts - _Geothermal, Hydro power, solar power, and wind power_
- Addition: _Sequestered Commodity_ as resources that become commodities when captured rather than extracted. 
   - Addition:  Sequestered Commodity child concept _Greenhouse Gases_  

FYI - @lisa-woody @DavidCrosswellGSQ

### CODE REVIEW

File: vocabularies/geo-commodities.ttl [hunk 1]
```diff
@@ -2236,6 +2266,38 @@ cdt:ferrous-metal a skos:Concept ;
...
+    skos:inScheme <http://linked.data.gov.au/def/geo-commodities> ;
+    skos:altLabel "energy resources"@en, 
+        "renewable energy resources"@en ;
+    skos:prefLabel "non-physical resources"@en .
+
+cdt:geothermal-energy a skos:Concept ;
+    skos:broader cdt:energy-resources ;
+    skos:definition "Thermal energy generated and stored within the Earth"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/geo-commodities> ;
+    skos:prefLabel "geothermal energy"@en .
```
Comment (talebib) [inline vocabularies/geo-commodities.ttl:2281]: All good.
Fine by me.

File: vocabularies/geo-commodities.ttl [hunk 2]
```diff
@@ -2340,3 +2402,18 @@ cdt:industrial-mineral a skos:Concept ;
...
+    skos:topConceptOf <http://linked.data.gov.au/def/geo-commodities> .
+
+cdt:greenhouse-gases a skos:Concept ;
+    skos:broader cdt:sequestered-commodity ;
+    skos:definition "Greenhouse gases that are an economic resource when captured for storage and sequestration."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/geo-commodities> ;
+    skos:altLabel "GHG"@en,
+        "Carbon GeoStorage"@en,
+        "Carbon Capture"@en ;
+    skos:prefLabel "greenhouse gases"@en .
```
Comment (talebib) [inline vocabularies/geo-commodities.ttl:2419]: I suggest @predam1 to review GHG section for definition etc.

Comment (SalEdwards01) [inline vocabularies/geo-commodities.ttl:2419]: Looks good to me, but I agree Micaela should review the definition here.

**Comment by predam1** [comment]
change 'sequestered commodity' to 'sequestered CO2'
definition: Greenhouse gases (including CO2) adsorb and emit radiant energy, which is believed to cause the greenhouse effect on Earth

**Comment by KellyVance** [comment]
> change 'sequestered commodity' to 'sequestered CO2'
> definition: Greenhouse gases (including CO2) adsorb and emit radiant energy, which is believed to cause the greenhouse effect on Earth

As per original pull request comment
Sequestered Commodity is a top concept to refer to any commodity (current/real or future/hypothetical) that is valued when captured, rather than extracted.

I have updated the Greenhouse Gases concept definition to include the information suggested.

**Comment by predam1** [comment]
I understand why you would want a broader definition but this commodity is used in conjunction with '**Carbon**' geostorage. Other categories can be entered later on when needed.

**Comment by KellyVance** [comment]
> I understand why you would want a broader definition but this commodity is used in conjunction with '**Carbon**' geostorage. Other categories can be entered later on when needed.

That's exactly why there are two additions. The broader more generic type in line with the structure of the rest of the vocabulary that can accommodate other categories added later without having to alter the hierarchy, and the specific Greenhouse gas concept to meet our current requirement.

**Comment by predam1** [comment]
This means I don't understand the structure just by looking at the code. Sorry.

**Comment by KellyVance** [comment]
@SalEdwards01 @talebib @predam1 
Using the green Review Changes button (top right) can you please approve, comment, or request changes as needed to progress this request? Thanks.

**Pull request merged**